app/src/main/java/org/glucosio/android/DomainConstants.java

```diff
@@ -0,0 +1,5 @@
+package org.glucosio.android;
+
+public class DomainConstants {
```

Comment:
Could rename to UnitConstants, making more obvious what is it.

Reply:
`DomainConstants` is ok, because it says there would be primary final values (not only units). Your name says that there you would find first Units, then constants.

@emartynov I would do not change it

Reply:
Renamed and made some small refactoring